kubectl-direct-csi/cmd/add-drives.go

```diff
@@ -97,12 +102,14 @@ func (c *csiAddDrivesCmd) run(args []string) error {
 	}
 
 	nodeSet := set.CreateStringSet(nodes...)
-	if nodeSet != nil {
+	if !nodeSet.IsEmpty() {
 		for _, drive := range drives.Items {
 			if nodeSet.Contains(drive.OwnerNode) {
 				match, _ := regexp.Match(args[0], []byte(drive.Path))
 				if match {
 					drive.DirectCSIOwned = true
+					drive.RequestedFormat.Filesystem = c.fileSystem
+					drive.RequestedFormat.Force = c.force
```

Comment:
`RequestedFormat` also has `Mountpoint`. Aren't we using it here?

Reply:
Since we are not taking it from the user, we can assume mountpoint value for now.